### DISCUSSION TITLE
Send tokens (TIA) from Forma to Stride

### DISCUSSION THREAD

**Discussion by calhegman** [blockscout/blockscout#10969] [Support]
Hi. A few days ago I send TIA from Forma to Stride via bridge.forma..art. Then I sent TIA from Stride to Celestia via SwapFast, but the transaction failed and my tokens disappeared. Leap Wallet support told me, that my tokens haven't entered to Stride. How is it possible? In metamask and transfer history on Forma bridge site this transaction completed. Please, help me solve this problem.

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
